cmd/copy.go

```diff
@@ -72,6 +72,8 @@ type rawCopyCmdArgs struct {
 	exclude               string
 	includePath           string // NOTE: This gets handled like list-of-files! It may LOOK like a bug, but it is not.
 	excludePath           string
+	includeRegex	   	  string
```

Comment:
Hmmm... I am trying to think of why we _shouldn't_ just accept regex in the normal include and exclude filter.

Possibly because we use it in the blob traverser as a performance option... BUT, we already have a complexity limitation on it, which would completely mitigate this question.

@zezha-msft and @siminsavani-msft any thoughts as to why we couldn't merge these? I'm mostly just trying to reduce the number of flags we have.

Reply:
Not sure if combining the the regex with normal in/ex filter would work smoothly as a regex library is being used to match the files (zc_filter.go, ln 257). @t-cguruceaga-msft Could you comment more on this?